packages/cli/src/user-management/email/templates/user-invited.mjml

```diff
@@ -0,0 +1,18 @@
+<mjml>
+	<mj-include path="./_common.mjml" />
+	<mj-body>
+		<mj-section>
+			<mj-column>
+				<mj-text font-size="24px" color="#ff6f5c"> Welcome to n8n! 🎉 </mj-text>
+			</mj-column>
+		</mj-section>
+		<mj-section background-color="#FFFFFF" border="1px solid #ddd">
+			<mj-column>
+				<mj-text>You have been invited to join n8n: {{domain}}.</mj-text>
+				<mj-text>To accept, please click the button below.</mj-text>
+				<mj-button href="{{inviteAcceptUrl}}">Set up your n8n account</mj-button>
+			</mj-column>
+		</mj-section>
+		<mj-include path="./_logo.mjml" />
```

Comment:
With default settings on Gmail the logo doesn't render. Not sure if we'd like to include it anyway despite this.

![Capture 2024-08-26 at 10 03 46@2x](https://github.com/user-attachments/assets/e09f8da2-4f88-444b-8df9-1e7b4d1f0e2f)